typescript/infra/scripts/sealevel-helpers/print-multisig-ism-config.ts

```diff
@@ -0,0 +1,45 @@
+import { IsmType } from '@hyperlane-xyz/sdk';
+
+import { multisigIsms } from '../../config/multisigIsm.js';
+import { getChains } from '../../config/registry.js';
+import { getArgs, withContext } from '../agent-utils.js';
+
+// This script exists to print the default multisig ISM validator sets for a given environment
+// so they can easily be copied into the Sealevel tooling. :'(
+
+async function main() {
+  const args = await withContext(getArgs())
+    .describe('local', 'local chain')
+    .choices('local', getChains())
+    .demandOption('local').argv;
+
+  const config = multisigIsms(
+    args.environment,
+    args.local,
+    IsmType.MESSAGE_ID_MULTISIG,
+    args.context,
+  );
+
+  // Cap any thresholds to 4 due to the Sealevel transaction size limit.
+  // Any higher than 4 at the moment will cause warp route synthetic deliveries to fail.
+  // Example message Solana -> Eclipse that mints a synthetic:
+  // https://explorer.eclipse.xyz/tx/3wcMvqZZjQon9o8nD49e3ci16AUJopZRLAfsAfs16ZrxgoNLoboNvrbV1hQHbnN3KXrWSqHmKnmM28mUvh5Un5Hd/inspect.
+  // At the time, the Solana threshold was 3. Taking the max tx size of 1232 and the tx's size 1121,
+  // we can find the number of additional signatures to be: floor((1232 - 1121)/65) = floor(1.707) = 1.
+  // So the total number of signatures is 3 + 1 = 4.
+
+  const MAX_THRESHOLD = 4;
+
+  for (const chain of Object.keys(config)) {
+    if (config[chain].threshold > MAX_THRESHOLD) {
+      config[chain].threshold = MAX_THRESHOLD;
```

Comment:
nit: I normally use this command as `print-multisig-ism-config.ts > file.txt`. If others do too, we could use stderr to log that we're not using the configured threshold

Reply:
this is why I didn't add a log, but I didn't consider stderr, good call